util/dvsim/dvsim.py

```diff
@@ -486,9 +486,8 @@ def parse_args():
     seedg = parser.add_argument_group('Build / test seeds')
 
     seedg.add_argument("--build-seed",
-                       nargs="?",
                        type=int,
-                       const=random.getrandbits(32),
```

Comment:
This was intentional. If you do not pass `--build-seed`, this change will cause `args.build_seed` to be set to some value, as opposed to `None`, making each build potentially different, which is not expected if `--build-seed` is not passed.

Reply:
Ahh! Now I understand. I'll just drop this patch from the series: we can always take another look at the remaining uses afterwards.